src/postgres/values.yaml

```
@@ -0,0 +1,7 @@
+auth:
+  username: gitlab
+  database: ###ZARf_VAR_DB_NAME###
```

```suggestion
  database: ###ZARF_VAR_DB_NAME###
```
not sure if it matters but for consistency.